### DISCUSSION TITLE
Issues with loading on start up

### DISCUSSION THREAD

**Discussion by Looodon** [ramensoftware/windhawk-mods#895] [Q&A] [answered]
Hey!
1) Windhawk doesn't load on startup, should i add it to the "shell:startup" files? or is there another way?

2) when it runs on startup through the "shell:startup" method the icons load without the padding:
![Screen Shot (2024-08-23 20_20_02)](https://github.com/user-attachments/assets/d4d73463-c34a-4b10-ac68-a4cb08f9e30f)

I then have to go fix it every time, It only happens on the first time the app is launched. Turning it off and on doesn't do that.
![Screen Shot (2024-08-23 20_20_53)](https://github.com/user-attachments/assets/761a5e96-ea93-49c8-9bf0-e7f5c099eed5)

**Comment by m417z** [comment, marked as answer]
Hi,

1. Are you using the portable version? If not, then no need to add it to the Startup folder, it should launch on startup automatically.
2. That might happen if Windhawk starts with a delay, which can happen if it starts from the Startup folder. If you're using the portable version, I suggest to use the standard version instead.

**Reply by Looodon** [reply]
Yes, I'm using the portable version! I'll switch to the standard version then thanks for the response
